### ISSUE TITLE
Encouraging attendance and participation through an incentive system

### ISSUE THREAD

**Issue by zg** [rit-csc/constitution#8]
The Computer Science Community has been lacking, minus the e-board members, in attendance at most events and meetings. Christina has suggested that we add in an incentive system that will give new and existing members a stronger interest in being present at events and meetings. This system would have a web-based component which would allow members to view membership profiles, track progress within each tier, and see what kind of rewards would be available at each tier.

For example, a webpage would exist for a hypothetical CSC member named Alex, where they would be able to view what tier they're on, what would be required to unlock the next tier, and what kind of rewards would come with unlocking that tier.

I'd like to open this issue on our constitution so that we can bring ideas forward on the subject matter.

**Comment by khipkin** [comment]
Yeah, this has been discussed before, but we could never think of a system that would work.
